### DISCUSSION TITLE
[New Feature] Translucent tooltips

### DISCUSSION THREAD

**Discussion by Markgrafin** [Phobos-developers/Phobos#893] [Ideas and feedback]
I think tooltips can do better
The tooltip of Red Alert 3 is a good choice
I used some pictures to express what I want to say
![0V70E JIGZ8{EMDMDBR@@K](https://user-images.githubusercontent.com/81842744/113477036-e8282780-94b1-11eb-9d4e-6512953af875.png)
![7TRA}X{274NC18L`W0CI9UH](https://user-images.githubusercontent.com/81842744/113477037-e9f1eb00-94b1-11eb-852c-c2f9ae9f7ed5.png)
![CJR IGH}(HR_5T7B79F4(18](https://user-images.githubusercontent.com/81842744/113477039-ea8a8180-94b1-11eb-8944-b259c3ffa349.png)
![O6~ZN{A4%5@K9 FZ$2EHP12](https://user-images.githubusercontent.com/81842744/113477040-ebbbae80-94b1-11eb-8895-1b1bcb11beef.png)
![XB }E %ZA`I)XLWG69825AO](https://user-images.githubusercontent.com/81842744/113477043-ececdb80-94b1-11eb-9e49-e856f957fe84.png)
![$7FH8AIXURG{ )`MBARJGM1](https://user-images.githubusercontent.com/81842744/113477045-ee1e0880-94b1-11eb-8572-2c7353121efc.png)

**Comment by AutoGavy** [comment]
yes, need to fix translucent csf background.

**Comment by Markgrafin** [comment]
![CJR IGH}(HR_5T7B79F4(18](https://user-images.githubusercontent.com/81842744/113477129-7d2b2080-94b2-11eb-84d1-26ec83d18284.png)
![7TRA}X{274NC18L`W0CI9UH](https://user-images.githubusercontent.com/81842744/113477137-82886b00-94b2-11eb-8338-d88c11df5ba1.png)
![O6~ZN{A4%5@K9 FZ$2EHP12](https://user-images.githubusercontent.com/81842744/113477150-8fa55a00-94b2-11eb-81be-9dfe71d1c1e2.png)
All pictures]

**Comment by Metadorius** [comment]
What exactly do you suggest? Build time? That is dependant on Ares but is planned. Different accent colors weren't yet investigated. Transparency... dunno on that either.

**Comment by secsome** [comment]
I think transparency need to be fixed

**Comment by Metadorius** [comment]
Weren't tooltips always opaque?

**Comment by secsome** [comment]
Not quite possible for now. But leave it open for further investigation.
